packages/amplify-category-api/src/provider-utils/awscloudformation/container-artifacts-graphql.js

```diff
@@ -0,0 +1,132 @@
+export const containerFiles = {
+    Dockerfile: `FROM node:alpine
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+
+EXPOSE 3000
+
+CMD ["node", "index.js"]
+    `,
+    'index.js': `'use strict';
+
+const PORT = 3000;
+
+var express = require('express');
+var { graphqlHTTP } = require('express-graphql');
+var { buildSchema } = require('graphql');
+var Query = require('./schema.js');
+var root = require('./resolvers.js');
+
+var app = express();
+
+// Enable CORS for all methods
+app.use(function (req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*")
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
+  next()
+});
+
```

Comment:
Should we include graphiql and expose and endpoint with graphiql

Reply:
It will use graphiql see the `graphiql: true` passed into `app.use` below